Makefile

```
@@ -79,6 +79,7 @@ help:
 	@echo 'Targets:'
 	@echo '  all          - download dependencies and compile telegraf binary'
 	@echo '  deps         - download dependencies'
+	@echo '  docs         - generate documentation and embed sample-configurations into READMEs'
```

Which 'documentation' does this command generate?

---

None yet, but I thought if I give it that general name I already account for future changes... :-)

---

It doesn't do that, so I wouldn't add it in yet. IMHO
```suggestion
	@echo '  docs         - embed sample-configurations into READMEs'
```